src/open-in-gitpod.ts

```diff
@@ -0,0 +1,9 @@
+import { renderGitpodUrl } from "./utils";
+import { ConfigProvider } from "./config";
+
+async function run() {
```

Comment:
IMO if we managed to get rid of the `async` here Firefox would stop flagging it as popup.

Reply:
Thanks for the hint. Unfortunately, I don't have any idea how to do this at the moment. Would like to skip this for now.